src/electron/platform/android/rule-information.ts

```diff
@@ -12,12 +12,12 @@ export class RuleInformation {
     constructor(
         readonly ruleId: string,
         readonly ruleDescription: string,
-        readonly getUnifiedResolutionDelegate: GetUnifiedResolutionDelegate,
+        readonly getUnifiedFormattableResolutionDelegate: GetUnifiedResolutionDelegate,
```

Comment:
Should `GetUnifiedResolutionDelegate` be renamed to `GetUnifiedFormattableResolutionDelegate` ?